parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -190,6 +192,12 @@ downloadAzureCNI() {
     retrycmd_get_tarball 120 5 "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" ${VNET_CNI_PLUGINS_URL} || exit $ERR_CNI_DOWNLOAD_TIMEOUT
 }
 
+downloadOras() {
```

Comment:
this was already taken care of in: https://github.com/Azure/AgentBaker/pull/4691, wasn't it?

@AlisonB319